examples/wgpu-html5/example.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+let Example = {
```

Comment:
Use the simpler fetch API to read files.

Reply:
Done. Thanks, I wasn't aware of the `fetch` API.